client/webpack.client.base.config.js

```diff
@@ -45,13 +51,19 @@ module.exports = {
       './app/bundles/comments/startup/clientRegistration',
     ],
   },
+
+  output: {
+    filename: '[name]-bundle.js',
+    path: resolve('..', paths.output, paths.assets),
+    pathinfo: true,
+  },
+
   resolve: {
     extensions: ['.js', '.jsx'],
-    alias: {
-      libs: path.resolve(__dirname, 'app/libs'),
-      react: path.resolve(__dirname, 'node_modules/react'),
-      'react-dom': path.resolve(__dirname, 'node_modules/react-dom'),
-    },
+    modules: [
+      paths.source,
+      paths.node_modules,
```

Comment:
@robwise, Webpack guru, what should we change this to? No need to put this in paths.yml, right? per https://github.com/shakacode/react_on_rails/pull/834